recipe/meta.yaml

```diff
@@ -13,36 +13,42 @@ source:
 
 build:
   number: 0
+  skip: True  # [py2k]
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
-  build:
+  host:
     - python
     - pip
+    - pytest-runner
   run:
     - python
-    - setuptools
```

Comment:
Think we need this as they use [`pkg_resources` a few places]( https://github.com/PyCQA/astroid/search?p=1&q=pkg_resources&unscoped_q=pkg_resources ). Admittedly most of those are tests, but a few are not.